src/main/resources/assets/minebound/models/block/myrial_glass_block.json

```diff
@@ -1,6 +1,6 @@
 {
   "parent": "minecraft:block/cube_all",
   "textures": {
-    "all": "minecraft:block/myrial_glass"
+    "all": "minebound:block/myrial_glass_block"
```

Comment:
Do we want to rename block textures like this?

Reply:
I think just doing it for glass is fine.

Reply:
Since minecraft calls it glass_block

Reply:
The ID of glass block is actually just `minecraft:glass`, might want to follow that convention.

Reply:
The texture and model files are also just glass; datapack entries don't tend to specify the classification because the folders already do. It's only backing objects that specify the classification (ArmorItem, GlassBlock, TorchBlock, etc)

Reply:
I agree with Mason and Matt, the texture and data name for the block should be myrial_glass "Myrial Glass" to match the minecraft spec